pallenec

```diff
@@ -28,10 +34,24 @@ local function compile(in_ext, out_ext)
     end
 end
 
+local function compile_internal(filename, stop_after)
+    local ok, errs = driver.compile_internal(filename, stop_after, true)
+    if not ok then
+        util.abort(table.concat(errs, "\n"))
+    end
+end
```

Comment:
Couldn't you add the `inspect` call inside this function instead of adding a new argument?

Reply:
Sure. I thought of doing it this way, but did not know which was best, requiring `inspect` in pallenec or in the driver. I chose the driver to keep the pallenec script cleaner.

Reply:
I think it is better to add to pallenec because it will only be used there. Also, there will be less code repetiton.

Reply:
Ok, I'll prepare this change.